### ISSUE TITLE
Ctrl+W does closes entire window or previous tab, not just the history tab.

### ISSUE THREAD

**Issue by waynebloss** [DonJayamanne/gitHistoryVSCode#421]
When you press `Ctrl+W` in VSCode it's supposed to close the active tab. However this does not work for the Git History log window.

Instead, when you only have one tab open showing the Git History and you hit `Ctrl+W` - the entire VSCode window closes.

Alternatively, when you have more than one tab open but Git History is the active tab, when you hit `Ctrl+W` - the previously active tab closes along with the Git History tab.

**Comment by DonJayamanne** [comment]
Thanks for reporting this issue. Unfortunately I'm unable to replicate this at my end.
Please could you provide the following info:
* Extension version
* VS Code version
* OS

**Comment by waynebloss** [comment]
Extension version: **0.4.12**

**Comment by waynebloss** [comment]
This happens on both of my LInux / XFCE / Manjaro workstations. I'll try it on Windows as soon as I can.

**Comment by waynebloss** [comment]
Oddly enough, I updated the extension to 0.4.12 in one instance of VS Code and the problem seemed to go away. Then I tried in the other instances and it remained. After I restarted all instances of VS Code, it was back.

I will investigate more, compare to my Mac and Windows machines or possibly caused just by my VS Code settings.

**Comment by waynebloss** [comment]
Closing this until I get a chance to investigate more.

**Comment by waynebloss** [comment]
I found how to reproduce this: Put focus on the VS Code titlebar first.

When you do this while a file tab is open, `Ctrl+W` closes it properly.

However, when you do this while Git History is the last active tab that is still open, `Ctrl+W` causes one of two bugs:

1. If Git History is the only tab open, the entire VS Code instance is closed.
2. If Git History is not the only tab open, the next most recently active tab gets closed along with Git History.

It seems as though the extension tab is supposed to end processing of `Ctrl+W` somehow and it doesn't do that in this case.

I've also compared this behavior to some other non-code windows such as: Settings. Details page for an Extension. In each case, even if I click on the VS Code titlebar before hitting `Ctrl+W` - the correct tab closes.

However, as I type this I decide to try one more: **Release notes.** This one exhibits the _same behavior_ as Git History.

So far, I've only tested this on my Linux (XFCE) desktop. I bet this is really a VS Code issue and that your extension doesn't even manage anything surrounding window/active tab management. So, I'll just leave my notes here for now.

P.S. I don't have any other extensions that create their own document tab to test against...
